packages/block-editor/src/components/global-styles/style.scss

```diff
@@ -134,6 +134,22 @@
 			box-shadow: inset 0 0 0 var(--wp-admin-border-width-focus) var(--wp-admin-theme-color);
 		}
 	}
+
+	&.is-uploading {
+		opacity: 0.5;
```

Comment:
Just out of curiosity (and maybe a question for a designer!), but do we have a consistent level of opacity that we usually set for loading states? At a quick glance, I wasn't able to find much other than `0.2` here: https://github.com/WordPress/gutenberg/blob/6f0037c6accf4b932d607806a16d7e963a6de915/packages/block-editor/src/components/link-control/style.scss#L142

I'm not a designer, but `0.5` in this case is looking visually pleasing to me.

Reply:
I also looked around for a consistent color/background/alpha but couldn't find one, hence the random `0.5`.

If there's one, I'm keen to know! 🙇🏻

Reply:
Maybe we should hide the button label and display the snipper, like the Featured image component at the top. Then, we can remove the opacity.

![CleanShot 2024-08-16 at 08 46 16](https://github.com/user-attachments/assets/72457157-9bbf-45c3-8723-93490ca021fe)

Reply:
> Maybe we should hide the button label and display the snipper, like the Featured image component at the top.

That would be a lot easier!

The placeholder background comes with a `#fff` background so I'll just switch off opacity.

Reply:
Done!

<img width="268" alt="Screenshot 2024-08-16 at 3 01 00 PM" src="https://github.com/user-attachments/assets/4e71e710-f538-4f2a-8e95-ad76eb8ff68f">